consensus/src/round_manager.rs

```diff
@@ -621,11 +621,12 @@ impl RoundManager {
     /// 2) call process_certificates(), which will start a new round in return.
     async fn process_vote(&mut self, vote: &Vote) -> anyhow::Result<()> {
         let round = vote.vote_data().proposed().round();
-        debug!(
+        info!(
             self.new_log(LogEvent::ReceiveVote)
                 .remote_peer(vote.author()),
-            "{}", vote
+            vote_data = vote,
```

Comment:
no variable-length field here, we should expect as many such logs as the received proposal log here: https://github.com/libra/libra/pull/6576 (so N per round, for N validators) **unless we're timing out**. In rounds that timeout, that event should be observed N times more (so NxN per round, for N validators).

Reply:
How big is the `Vote` object?  I'm pretty sure the `LedgerInfo` part of it is pretty big?
https://github.com/libra/libra/blob/master/consensus/consensus-types/src/vote.rs#L18-L30

Reply:
So, at minimum it's (N^2 + N) * the size of the block?

Reply:
<img width="508" alt="Screen Shot 2020-10-27 at 12 02 09 PM" src="https://user-images.githubusercontent.com/1316043/97349077-3fafd080-184c-11eb-9c1c-01d4da0fb9c6.png">

Reply:
A `Vote` is basically 3 `BlockInfo`s and some other fields. 
A `BlockInfo` is ~100 bytes if there is no reconfiguration, Nx32 bytes more if there's a reconfiguration.
So that's around 400 bytes per `Vote` when no reconfiguration is occurring, and much more if there's a reconfiguration.

We can take the same strategy that I'm proposing with the ReceiveProposal and only log the signa (epoch, round, id, executed_state_id, author).l, but not the proof, which would bring that down to ~100 bytes

Reply:
Yeah, just trying to keep it under ~1024 bytes if possible, if we're constantly logging it at this rate.  Over optimizing to less than 100 bytes is not too important

Reply:
So the current log line, logs any where from 30-300 in a second, averaging usually at the 60 mark [here](https://85bd0f3d76314c71869bce3ae260f6d2.us-west1.gcp.cloud.es.io:9243/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2020-10-29T20:15:54.357Z',to:'2020-10-29T20:23:01.365Z'))&_a=(columns:!(_source),filters:!(('$state':(store:appState),meta:(alias:!n,disabled:!f,index:'44a07b90-f794-11ea-af4e-cf9e04c7cc93',key:location.keyword,negate:!f,params:(query:'consensus%2Fsrc%2Fround_manager.rs:624'),type:phrase),query:(match_phrase:(location.keyword:'consensus%2Fsrc%2Fround_manager.rs:624')))),index:'44a07b90-f794-11ea-af4e-cf9e04c7cc93',interval:s,query:(language:kuery,query:''),sort:!()))

Reply:
for 30 validators I predicted 90/s without timeout and 900/round with timeout (not sure how long a timeout is), so that's about right! Forgot we were already logging these in debug :)

Reply:
so I realize that the log already truncates a number of fields via the use of custom Display implementation, so to avoid losing any data I instead add some new data in the schema and keep what we've been logging so far.